source/globalCommands.py

```diff
@@ -1361,10 +1361,13 @@ def script_reportStatusLine(self,gesture):
 			return
 		if scriptHandler.getLastScriptRepeatCount()==0:
 			ui.message(text)
-		else:
+		elif scriptHandler.getLastScriptRepeatCount()==1:
 			speech.speakSpelling(text)
+		else:
+			if api.copyToClip(text):
+				ui.message(_("%s copied to clipboard")%text)

```

Comment:
You need a translators comment above this line. Something like
#Translators: The message presented when the status bar is copied to the clipboard.